sharc/post_processor.py

```diff
@@ -457,6 +492,20 @@ def generate_ccdf_plots_from_results(
                         name=f"{legend}",
                     ),
                 )
+                # A trick to plog semi-logy plots with better scientific aspect.
+                # I should have left it to the user to decide if they want logy or not,
+                # but I think it is better to have it by default.
+                if logy:
+                    fig.update_yaxes(type="log")
+                    yticks = []
+                    n_right_zeros = -int(np.floor(np.log10(cutoff_percentage)))
+                    for i in range(n_right_zeros, 0, -1):
+                        for j in range(1, 10):
+                            yticks.append(j * (10**(-i)))
+                    yticks = yticks + [1.0]
+                    major_yticks = [10**(-i) for i in range(4)]
```

Comment:
Shouldn't major ticks be until the user defined limit?
```suggestion
                    major_yticks = [10**(-i) for i in range(int(np.log10(cutoff_percentage)))]
```

Reply:
Done!